internal/pkg/scaffold/helm/role.go

```diff
@@ -41,11 +40,12 @@ import (
 // CreateRoleScaffold generates a role scaffold from the provided helm chart. It
 // renders a release manifest using the chart's default values and uses the Kubernetes
 // discovery API to lookup each resource in the resulting manifest.
-func CreateRoleScaffold(cfg *rest.Config, chart *chart.Chart, isClusterScoped bool) (*scaffold.Role, error) {
+// The role scaffold will have IsClusterScoped=true if the chart lists cluster scoped resources
+func CreateRoleScaffold(cfg *rest.Config, chart *chart.Chart) (*scaffold.Role, error) {
 	log.Info("Generating RBAC rules")
 
 	roleScaffold := &scaffold.Role{
-		IsClusterScoped:  isClusterScoped,
+		IsClusterScoped:  false,
```

Comment:
Any reason why we couldn't just remove this as well?

Reply:
@LiliC  I see what you mean. The default is false but I just wanted to make the default setting explicit for the helm operator since it’s the only one that needs to set this option.